packages/analytics-js/src/components/capabilitiesManager/detection/adBlockers.ts

```diff
@@ -1,28 +1,35 @@
-const isAdBlockedElement = (element: HTMLScriptElement): boolean => element.id === 'ad-block';
+import { IErrorHandler } from '@rudderstack/analytics-js/services/ErrorHandler/types';
+import { HttpClient } from '@rudderstack/analytics-js/services/HttpClient/HttpClient';
+import { ILogger } from '@rudderstack/analytics-js/services/Logger/types';
+import { state } from '@rudderstack/analytics-js/state';
 
-/**
- * Utility method to handle script load errors that are ad blocked
- */
-const handleScriptLoadAdBlocked = (
-  errorMessage: string,
-  element: HTMLScriptElement,
-  analyticsInstance?: any,
-): string => {
-  // SDK triggered ad-blocker script
-  if (isAdBlockedElement(element)) {
-    // Track ad blocked page in analytics
-    analyticsInstance?.page(
-      'RudderJS-Initiated',
-      'ad-block page request',
-      { path: '/ad-blocked', title: errorMessage },
-      analyticsInstance?.sendAdblockPageOptions,
-    );
+const detectAdBlockers = (errorHandler?: IErrorHandler, logger?: ILogger): void => {
+  // Apparently, '?view=ad' is a query param that is blocked by majority of adblockers
 
-    // No need to proceed further for Ad-block errors
-    return '';
-  }
+  // Use source config URL here as it is very unlikely to be blocked by adblockers
+  // Only the extra query param should make it vulnerable to adblockers
+  // This will work even if the users proxies it.
+  // The edge case where this doesn't work is when HEAD method is not allowed by the server (user's)
+  const baseUrl = new URL(state.lifecycle.sourceConfigUrl.value as string);
+  const url = `${baseUrl.origin}${baseUrl.pathname}?view=ad`;
```

Comment:
can you please double check the lists i added in the ticket to see if view=ad exists in both lists or we need to add more params here?

Reply:
This is a very prominent way to fetch ads. I've verified that this request gets blocked by more than 90% of the ad blockers.

What are the lists you are talking about? I don't see any in the ticket.

Reply:
added links in the comments section

Reply:
Thanks. I've added more details in the comment section of the card.